package.json

```diff
@@ -24,7 +24,7 @@
     "src/*"
   ],
   "dependencies": {
-    "react-native-linear-gradient": "git+https://github.com/wordpress-mobile/react-native-linear-gradient.git#v2.5.6-wp-1",
+    "react-native-linear-gradient": "https://raw.githubusercontent.com/wordpress-mobile/react-native-linear-gradient/172ade59e041e76bfceb87839f998663ea2f9d8d/react-native-linear-gradient-2.5.6-wp-1.tgz",
```

Comment:
In the future, it would be great if we could point to the tag in the URL instead of the commit hash.